packages/@ourworldindata/grapher/src/stackedCharts/StackedUtils.ts

```diff
@@ -28,6 +28,30 @@ export const stackSeries = <PositionType extends StackedPointPositionType>(
     return seriesArr
 }
 
+// This method shifts up positive y values and shifts down negative y values of a Series with Points in place.
+export const stackSeriesInBothDirections = <
```

Comment:
We could also replace `stackSeries` with `stackSeriesInBothDirections` (`stackSeriesInBothDirections` can do everything that `stackSeries` can do)

Reply:
But then stacked area charts would break in weird ways probably, right? Unless we explicitly set all negative values to zero beforehand.

Reply:
yap, that's right